packages/.coderabbit.yaml

```
@@ -0,0 +1,196 @@
+                      - Ensure descriptive test names convey the intent of each test.
+                      - Validate adherence to the Mocha/Chai/Jest test library best practices.
+               - Be concise and avoid overly nitpicky feedback outside of these best practices.
+        - path: "**/*.js"
+          instructions: |2-
+                Review JavaScript files for:
+                - Compliance with the Google JavaScript Style Guide.
+                - Dependency management without caret versions.
+         - Be concise and avoid overly nitpicky feedback outside of these best practices.
+        - path: "**/errors/**/*.ts"
+          instructions: |2-
+         Ensure custom error classes:
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 9

(indentation)